doc/nrf/device_guides/working_with_nrf/nrf91/nrf91_features.rst

```diff
@@ -365,39 +362,15 @@ Following is a list of the samples and applications with some information about
 * The :ref:`gnss_sample` sample does not use assistance by default but can be configured to use nRF Cloud A-GNSS, P-GPS, or a combination of both.
   The sample displays tracking and fix information as well as NMEA strings in the serial console.
 
-.. _nrf9160_gps_lte:
-
-Concurrent GNSS and LTE
-=======================
-
-An nRF9160-based device supports GNSS in LTE-M and NB-IoT.
-Concurrent operation of GNSS with optional power-saving features, such as extended Discontinuous Reception (eDRX) and Power Saving Mode (PSM), is also supported, and recommended.
-
-The following figure shows how the data transfer occurs in an nRF9160-based device with power-saving in place.
-
-.. figure:: images/power_consumption.png
-   :alt: Power consumption
-
-See `Energy efficiency`_ for more information.
-
-Asset Tracker enables the concurrent working of GNSS and LTE in eDRX and PSM modes when the device is in `RRC idle mode <Radio Resource Control_>`_.
-The time between the transition of a device from RRC connected mode (data transfer mode) to RRC idle mode is dependent on the network.
-Typically, the time ranges between 5 seconds to 70 seconds after the last data transfer on LTE.
-Sensor and GNSS data are sent to the cloud only during the data transfer phase.
-
 .. _nrf9160_ug_band_lock:
 
 Band lock
 *********
 
-The band lock is a functionality of the application that lets you send an AT command to the modem instructing it to operate only on specific bands.
-The band lock is handled by the LTE Link Control driver.
-By default, the functionality is disabled in the driver's Kconfig file.
-
-The modem can operate in the following E-UTRA Bands: 1, 2, 3, 4, 5, 8, 12, 13, 17, 18, 19, 20, 25, 26, 28, and 66.
-To check which bands are supported for a particular modem firmware version, see the `nRF9160 product website (compatible downloads)`_.
+The modem can operate on a number of LTE bands.
+To check which bands are supported by a particular modem firmware version, see the `nRF9160 product website (compatible downloads)`_.
```

Comment:
Do you know which part of the linked section has the information? I couldn't find anything on a quick look, so I would guess the same holds true for our users as well. Might as well improve the information on this part as well while we're working on the page.

Reply:
The supported bands are listed in modem firmware release notes, which are inside the zip files. It's not possible to link directly to the information, but we can improve the text to be more specific. I'll try to come up with something.